default/python/universe_generation/planets.py

```diff
@@ -4,27 +4,26 @@
 import util
 import universe_tables as tables
 
-
```

Comment:
There is not special rules for that case.  I think I made dis because comment on next lien. Will revert.